tests/contract-playground/src/DomainSeparatorCollision.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```

Comment:
Lower this version to one that accepts two functions with the same sig:
```
withdraw(uint256)
OwnerTransferV7b711143(uint256)
```

source: https://www.reddit.com/r/ethdev/comments/6n6t9w/function_hash_collision_potential_misuse/